src/Maker/DddMaker.php

```diff
@@ -134,6 +150,36 @@ protected function getDomainConfigPath (string $domain) : string
         return $locator->locate("domain_config.yaml");
     }
 
+    /**
+     * Returns the locations of all domain configuration file.
+     *
+     * @throws FileLocatorFileNotFoundException If the file cannot be found
+     */
+    protected function getAllDomains () : array
+    {
+        $domains = [];
+        $finder = (new Finder())
+            ->in('src/')
+            ->name('domain_config.yaml');
+
+        foreach ($finder->getIterator() as $item)
+        {
+            $domain = \explode("/", "{$item}")[1];
+            $domains[] = $domain;
+        }
+
+        return $domains;
+    }
+
+
+    /**
+     * If this method returns true the command will ask the user to select a domain from a list of domains.
+     * If this method returns false the command will ask the user to provide the name of the domain as a string.
+     */
+    protected function askToSelectDomain () : bool {
```

Comment:
If the CI would have been set up, then the PhpCsFixer would have caught this :)

```suggestion
    protected function askToSelectDomain () : bool
    {
```